### PR TITLE
Project JS Tic Tac Toe, ready for review

### PR DESCRIPTION
**Pull request by noomdalv**
Project functionalities
1. Players can enter their name and start the game
2. Players can choose positions to mark them with their simbol ( X || O)
3. Game checks for a winner if a combination is found
4. Game checks for draw if all positions are occupied without matching combinations

### CODE REVIEW

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED

Hi Team good job on this one :clap: :clap: the UI looks awesome and code is neat too :slightly_smiling_face:  please see my suggestions below 

- make changes to the eslintrc.json file see this link for what should be there [see this](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)

- [OPTIONAL] divide logic into different files for better organization, so gameboard can  be in one file, the game logic can be in another.

**please make changes and request a re-review** :+1: 

**RAPHAEL NORIODE** :sunglasses:

**Comment by noomdalv** [comment]
Changes applied:

- .eslintrc.json file was updated
- eslint linter should now work properly with the correct config

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+		"no-plusplus": "off",
+		"no-tabs": "off"
+  }
+}
```
Comment (bolah2009) [inline .eslintrc.json:19]: Please let's stick with the same config file on https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json.

> Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))

**Review by bolah2009 — APPROVED** [review]
It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Good luck with your next milestones. :star2: 

Please see the comment below before merging.

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**